core/src/main/scala/scalacss/mutable/Register.scala

```diff
@@ -130,13 +130,21 @@ final class Register(initNameGen: NameGen, macroName: MacroName, errHandler: Err
     implicit def f[W, I: StyleLookup](implicit h: ClassNameHint): Case.Aux[Named[W, StyleF[I]], Named[W, I => StyleA]] = at(_ map registerF[I])
   }
 
+  def registerKeyframes(keyframes: Keyframes)(implicit cnh: ClassNameHint): Keyframes = {
+    val cn = macroName(cnh, keyframes.name.value) map ensureUnique
+    val kf = keyframes.copy(name = cn.get)
+    _keyframes :+= kf
+    kf
+  }
+
   def styles: Vector[StyleA] = mutex {
     _rendered = true
     _styles
   }
 
-  def css(implicit env: Env): Css =
-    Css(styles)
+  def css(implicit env: Env): Css = {
+    Css.prepareStyles(styles) ++ Css.prepareKeyframes(_keyframes)
+  }
```

Comment:
FYI I'm not a fan of surrounding single, pure expressions in `{ ... }`. They allow accidental side-effects, and broken/split expressions to go accidental unnoticed (unless `-Ywarn-discard` is on, which it isn't yet but should be).